models/stg_iterable__event.sql

```diff
@@ -28,27 +28,33 @@ fields as (
 
 final as (
     
-    select 
-        _fivetran_id as event_id,
-        campaign_id,
-        content_id,
+    select
+        cast(_fivetran_id as {{ dbt.type_string() }} ) as event_id,
+        cast(campaign_id as {{ dbt.type_string() }} ) as campaign_id,
+        cast(content_id as {{ dbt.type_string() }} ) as content_id,
         created_at,
         cast( {{ dbt.date_trunc('day', 'created_at') }} as date) as created_on,
         lower(email) as email,
         additional_properties,
         event_name,
-        message_bus_id,
-        message_id,
-        message_type_id,
+        cast(message_bus_id as {{ dbt.type_string() }} ) as message_bus_id,
+        cast(message_id as {{ dbt.type_string() }} ) as message_id,
+        cast(message_type_id as {{ dbt.type_string() }} ) as message_type_id,
         recipient_state,
         status,
         transactional_data,
         unsub_source,
         user_agent,
         user_agent_device,
-        _fivetran_synced
+        _fivetran_synced,
+        cast(_fivetran_user_id as {{ dbt.type_string() }} ) as _fivetran_user_id
+
+        {{ fivetran_utils.fill_pass_through_columns('iterable_event_pass_through_columns') }}
```

Comment:
Were the passthrough variables tested? I tried the following passthrough config for both the event and user_history staging models and ran into the following error.

![image](https://github.com/fivetran/dbt_iterable_source/assets/74217849/e1391a46-342f-48de-9097-b341b93463af)

Reply:
The passthrough columns have been fixed and tested